src/components/side-bar/index.js

```diff
@@ -0,0 +1,116 @@
+import BaseComponent from "../base-component.js";
+import FiltersList from "../filters-list/index.js";
+import DoubleSlider from "../double-slider/index.js";
+
+import "./side-bar-style.css";
+
+export default class SideBar extends BaseComponent {
+  components = {};
+
+  constructor(categoriesFilter = [], brandFilter = []) {
+    super();
+    this.categoriesFilter = categoriesFilter;
+    this.brandFilter = brandFilter;
+
+    this.initializeComponents();
+    this.init();
+    this.renderComponents();
+    this.addEventListeners();
+  }
```

Comment:
**Code Structure Issue**: The constructor of SideBar component is overloaded with multiple initializations and direct DOM interactions. <br> **Fix**: Simplify the constructor by separating concerns into well-defined methods and avoiding direct DOM interactions. <br> **Code Suggestion**: 
 ```
 - this.initializeComponents();
 - this.init();
 - this.renderComponents();
 - this.addEventListeners();
 + this.setupComponents();
 ```